api/proto/teleport/machineid/v1/bot_instance.proto

```diff
@@ -101,11 +97,9 @@ message BotInstanceStatusAuthentication {
   // method, this counter is checked during renewal and the Bot is locked out if
   // the counter in the certificate does not match the counter of the last
   // authentication.
-  int32 generation = 5;
```

Comment:
This is a breaking proto change, I _think_ we can get away with this because there's no real consumers of this API yet, but if we're worried about it we can either keep this as an int32 or swap this for a v2 field and do it "properly".

I can also mark the removed fields as reserved, in that case.

Reply:
Unfortunately - the linter can't be overriden 😓  You'll probably need to just roll to a new field name and number

Reply:
Ah, always forget about the linter. It's harmless enough to mark the removed fields as `reserved`, but I think I'll drop the type change. Some napkin math suggests the smaller type really won't cause a problem:
```
In [1]: 2147483647 / (3 * 24 * 365)
Out[1]: 81715.51168188736
```

I figure ~82 thousand years of continuous renewals ought to be enough for anybody 🤷 

(Oddly, the proto language guide it links to explicitly says `int32` and `uint64` are compatible, but the linter complains about it anyway, oh well.)

Reply:
Hah, I think 82 thousand ought to be enough for anyone 😉